### PR TITLE
Add webpack react description

### PR DESCRIPTION
**Pull request by imranhossainemi**
### Hello ror react app project update:
- init commit with jsbundling-rails gem , webpack & react
- Add linters files
- add readme file
- install needed node dependencies 
- Create a babel.config.js in the root folder
- Update webpack.config.js to include the babel loader
- Add CRUD for message function
- Store data in postgresql
- Create an API endpoint that selects a random greeting
- Create a static view with random greeting
- For manual injection of greetings run this command: rails db:seed

### CODE REVIEW

File: app/controllers/messages_controller.rb [hunk 1]
```diff
@@ -1,31 +1,29 @@
...
+  def edit; end
 
   # POST /messages or /messages.json
   def create
     @message = Message.new(message_params)
 
     respond_to do |format|
       if @message.save
-        format.html { redirect_to message_url(@message), notice: "Message was successfully created." }
+        format.html { redirect_to message_url(@message), notice: 'Message was successfully created.' }
```
Comment (Davidosky007) [inline app/controllers/messages_controller.rb:26]: * Kindly create 5 different greetings as mentioned in the [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/connect-frontend-frameworks/rails_react_webpack.md#:~:text=Create%20a%20table%20for%20storing%20your%20messages%20and%20create%205%20different%20greetings)

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @imranhossainemi 👋 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* Well written PR 👍 
* Good looking readme 👍 
* Webpack setup properly 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/messages_controller.rb (hunk 1 above)
Comment (imranhossainemi) [reply app/controllers/messages_controller.rb:26]: @Davidosky007  WTF YOU ARE TALKING ABOUT , I have a given CRUD method add new greeting into database. You do not see that or I need to give you a manual injection key insert it into database.

**Review by Olamarx — APPROVED** [review]
Hi @imranhossainemi,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Prepopulated seed file ✔️
Webpack setup ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**